ambry-router/src/test/java/com/github/ambry/router/CryptoJobHandlerTest.java

```diff
@@ -73,6 +73,11 @@ public void before() {
 
   @After
   public void after() {
+    // Add delay in case buffers are released in background
+    try {
+      Thread.sleep(500);
```

Comment:
@justinlin-linkedin The test passes consistently (so far) with this sleep added. This is a temp fix.  A better one is to enhance the afterTest() to check the metrics repeatedly up to a timeout.

Reply:
Added the changes to NettyByteBufLeakHelper.afterTest() in this PR.